src/plugins/intel_gpu/src/runtime/execution_config.cpp

```diff
@@ -79,7 +79,8 @@ void ExecutionConfig::set_default() {
         std::make_tuple(ov::intel_gpu::allow_new_shape_infer, false),
         std::make_tuple(ov::intel_gpu::use_only_static_kernels_for_dynamic_shape, false),
         std::make_tuple(ov::intel_gpu::buffers_preallocation_ratio, 1.1f),
```

Comment:
[random spot] please add some tests

Reply:
What should they test specifically? There are no tests for traditional caching which I could use for reference. I wanted to compare constants' data directly in an original & imported models but I couldn't find a way to access that data using the standard API.

Reply:
You can find many test cases that end with '_cached' as below:
https://github.com/openvinotoolkit/openvino/blob/4758030e8e2c13383229c438aba945cdf86ad7a0/src/plugins/intel_gpu/tests/unit/test_cases/gemm_gpu_test.cpp#L3331-L3333
These tests are for the model caching feature.

Reply:
Tests added.